lib/rucio/rse/protocols/protocol.py

```diff
@@ -337,8 +338,10 @@ def _get_path(self, scope, name):
 
     def _get_path_nondeterministic_server(self, scope, name):  # pylint: disable=invalid-name
         """ Provides the path of a replica for non-deterministic sites. Will be assigned to get path by the __init__ method if neccessary. """
-        scope = InternalScope(scope)
+        vo = get_rse_vo(self.rse['id'])
+        scope = InternalScope(scope, vo=vo)
         rep = replica.get_replica(scope=scope, name=name, rse_id=self.rse['id'])
+        print(rep)
```

Comment:
Just want to check this is intentional (I know I add loads of rogue `print()` statements when debugging, and it doesn't look like we printed anything here before).

Reply:
It's not intentional, thanks for spotting